ipu-plugin/pkg/ipuplugin/ovsbridge.go

```diff
@@ -101,19 +107,19 @@ func (b *ovsBridge) EnsureBridgeExists() error {
 		return err
 	}
 	// Flush any existing IP addresses from the bridge interface from any previous runs
-	flushIPCmd := []string{"net", "exec", netNs, "ip", "addr", "flush", "dev", b.bridgeName}
-	if err := utils.ExecOsCommand("ip", flushIPCmd...); err != nil {
+	flushIPCmd := []string{"-t", netNs, "-n", "ip", "addr", "flush", "dev", b.bridgeName}
```

Comment:
what was the issue we faced with ip netns exec which led us to move to nsenter?

Reply:
ip netns doesn't work in the new way of doing things. It only accepts the netns and not the PID directly